earthmover/earthmover.py

```diff
@@ -314,6 +327,7 @@ def execute(self, subgraph):
                 if not node.data:
                     node.execute()  # Sets self.data in each node.
                     node.post_execute()
+                    self.metadata["row_counts"].update({'$'+node.type+'s.'+node.name: len(node.data)})

```

Comment:
I want to be _very_ careful here. By default, we do not execute nodes to get counts unless explicitly specified, and even then we wait until the very end to try to speed up runtime. I want us to add an if-statement that checks whether the results file is turned on in Earthmover before trying to get the size of the dataframe for the results file.